packages/mui-material/src/SpeedDial/SpeedDial.d.ts

```diff
@@ -83,15 +124,25 @@ export interface SpeedDialProps
    *   enter: theme.transitions.duration.enteringScreen,
    *   exit: theme.transitions.duration.leavingScreen,
    * }
+   * @deprecated
```

Comment:
We shouldn't deprecate `transitionDuration`

Reply:
ah yeah... my bad, wrong slot